src/test/resources/org/jenkinsci/plugins/docker/workflow/declarative/dockerPullLocalImage.groovy

```diff
@@ -46,7 +46,7 @@ pipeline {
         stage("in pulled image") {
             agent {
                 docker {
-                    image "maven:3-alpine"
+                    image "maven:3-jdk-8-slim"
```

Comment:
`3-alpine` hasn't been updated for around 5 years, and for whatever reason `mvn` hangs in the  container running from the pulled image (but not in the container running from the built image). I suspect it has something to do with QEMU emulation since I am running on an M1 Mac, but I really have no idea. I switched to `3-jdk-8-slim`, which is actively maintained and has native `arm64` builds, and the test passes locally now.